index.html

```diff
@@ -0,0 +1,82 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <meta name="description" content="">
+    <meta name="keywords" content="">
+    <title>Attr🔹ct Quiz</title>
+    <link rel="preconnect" href="https://fonts.gstatic.com">
+    <link href="index.css" rel="stylesheet">
+    <link rel="icon" href="images/cover.png">
+    <link href="https://attr-act.netlify.app/Attract.css" rel="stylesheet">
+</head>
+<body>
+    <header class="header">
+        <div class="header-left">
+            
+            <button class="secondary-link"><a href="index.html">Home</a></button>
+        </div>
+    <h1 class="headings">Attr🔹ct Quiz</h1>
+    <div class="header-right">
+        <a href="https://github.com/GuptaEesh" target="_blank" rel="noopener" rel="noreferrer"><img
+                class="img-xsm profile-avatar" src="https://cdn-icons-png.flaticon.com/512/25/25231.png"
+                alt="Github icon"></a>
+        <a href="https://twitter.com/ra1711003010629" target="_blank" rel="noopener" rel="noreferrer"><img
+                class="img-xsm profile-avatar" src="https://cdn-icons-png.flaticon.com/512/1051/1051382.png"
+                alt="Twitter icon"></a>
+        <a href="https://webdevjourney.hashnode.dev/" target="_blank" rel="noopener" rel="noreferrer"><img
+                class="img-xsm profile-avatar"
+                src="https://cdn.hashnode.com/res/hashnode/image/upload/v1611902473383/CDyAuTy75.png?auto=compress"
+                alt="hashnode icon"> </a>
+        <a href="https://www.linkedin.com/in/eesh-gupta-42673a144" target="_blank" rel="noopener"
+            rel="noreferrer"><img class="img-xsm profile-avatar"
+                src="https://cdn-icons-png.flaticon.com/512/174/174857.png" alt="Linkedin icon"> </a>
+    </div>
+</header>
+<main class="quiz-body">
```

Comment:
You can also section tag here.

Reply:
I have used the main tag as the content inside it is unique to this page.